context/context.go

```diff
@@ -139,23 +139,29 @@ func (ctx Context) GetInt64Param(i int) int64 {
 		panic(err)
 	}
 
-	return int64(param)
+	return param
 }
 
 func (ctx Context) Set(key interface{}, value interface{}) {
 	ctx.values[key] = value
 }
 
-func (ctx Context) Get(key interface{}) interface{} {
+func (ctx Context) Get(key interface{}, def ...interface{}) interface{} {
```

Comment:
Second thing - sooner o later, someone will try to pass more than one element here ;-)

Reply:
should I panic then? :)

Reply:
or break the backward compatibility by introducing a different behavior:

* add a separate set of methods, like `GetTypeXOrDefault()`, i.e. `GetBoolOrDefault()`
* in the original method, don't add the optional params list; just panic when the key is not found

Reply:
since the lib is on an early stage of development this shouldn't be very painfull

Reply:
I don't want to give too many functions to the user. I think I'll fail the test with the message

```
allowed to pass only 1 default value but %d got
```

I can see it's a quite common approach https://stackoverflow.com/questions/2032149/optional-parameters

Reply:
I noticed that golang tends to promote many practices that are questionable from the paradigms (both functional and OOP) perspective ;-) But time will show if this was a right way